config/application.rb

```diff
@@ -21,10 +21,7 @@
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
-
-# rubocop:disable Naming/ClassAndModuleCamelCase
```

Comment:
I don't agree with this change and the addition to the .rubocop.yml file. When renaming the project, the lines here were close to the problem. By moving them to .rubocop.yml, we practically guarantee that the rule won't be removed when the problem is fixed.